static/js/reducers/posts.js

```diff
@@ -0,0 +1,15 @@
+// @flow=
```

Comment:
are we planning to use `redux-hammock` for hitting these APIs? If so, we wont need these reducers.